governator-jetty/src/main/java/com/netflix/governator/guice/jetty/JettyModule.java

```diff
@@ -106,6 +113,31 @@ public JettyRunner(Server server, final LifecycleManager manager) {
                 throw new ProvisionException("Jetty server failed to start", e);
             }
         }
+        
+        public boolean isRunning() {
+            return server.isRunning();
+        }
+        
+        public int getLocalPort() {
+            return this.port;
+        }
+    }
+    
+    @Singleton
+    static class OptionalJettyConfig {
+        
+        @com.google.inject.Inject(optional=true)
+        private JettyConfig jettyConfig;
+        
+        public JettyConfig getJettyConfig() {
+            if(jettyConfig != null)
```

Comment:
Would much rather use the ternary operator here.  Or even Optional, since we're on Java8 now.
```
Optional.ofNullable(jettyConfig).orElse(() -> new DefaultJettyConfig());
```